src/o365/spo/commands/feature/feature-list.ts

```diff
@@ -0,0 +1,175 @@
+import auth from '../../SpoAuth';
+import { Auth } from '../../../../Auth';
+import config from '../../../../config';
+import * as request from 'request-promise-native';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption,
+  CommandValidate
+} from '../../../../Command';
+import SpoCommand from '../../SpoCommand';
+import { ContextInfo } from '../../spo';
+import Utils from '../../../../Utils';
+import { Feature } from './feature';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  url: string;
+  scope?: string;
+}
+
+class SpoFeatureListCommand extends SpoCommand {
+  public get name(): string {
+    return `${commands.FEATURE_LIST}`;
+  }
+
+  public get description(): string {
+    return 'Lists features for site or site collection';
```

Comment:
This description is ambiguous and it's not quite clear if retrieves features that are activated or that are available on the given scope. We should rename it to `Lists features activated on the specified site or site collection`.